server/README.md

```diff
@@ -1 +1,38 @@
-# Server Component
+# Twilio Voice React Native Reference App Server
+
+## What is this Project?
+
+The Twilio Voice React Native Reference App Server is a supplementary backend service for the Twilio Voice React Native Reference App. This Reference App serves as an example for developers who want to leverage the power of Twilio Programmable Voice in a React Native App targetting mobile platforms (iOS and Android).
+
+## Before you use this Project
+
+* Please ensure that you have the latest version LTS version of Node.js.
+* Yarn is also the preferred version manager for this project.
```

Comment:
No need yo mention that you prefer yarn. Just mention the version of yarn required to run this project.